code/modules/cargo/exports.dm

```diff
@@ -78,28 +78,12 @@ Then the player gets the profit from selling his own wasted time.
 	/// Types excluded from export
 	var/list/exclude_types = list()
 
-	/// cost includes elasticity, this does not.
-	var/init_cost
-
-
 
 /datum/export/New()
 	..()
-	SSprocessing.processing += src
-	init_cost = cost
 	export_types = typecacheof(export_types, only_root_path = !include_subtypes, ignore_root_path = FALSE)
 	exclude_types = typecacheof(exclude_types)
 
-/datum/export/Destroy()
-	SSprocessing.processing -= src
-	return ..()
-
-/datum/export/process()
-	..()
-	cost *= NUM_E**(k_elasticity * (1/30))
-	if(cost > init_cost)
-		cost = init_cost
-
 // Checks the cost. 0 cost items are skipped in export.
 /datum/export/proc/get_cost(obj/O, apply_elastic = TRUE)
```

Comment:
Can you fix this up, check for ``apply_elastic && k_elasticity!=0`` to avoid doing else twice for the same thing.